### ISSUE TITLE
[Windows] Camera usage cause application crash with AIR 32-bit when Logitech Capture installed

### ISSUE THREAD

**Issue by itlancer** [Gamua/Adobe-Runtime-Support#320]
## Problem Description
`Camera` usage cause application crash with AIR 32-bit when Logitech Capture installed.
There is no way to use `Camera` with 32-bit AIR applications in such cases.
Logitech Capture can be installed from official website: https://www.logitech.com/en-us/product/capture

It has been tested with AIR 32.0.0.89 and latest AIR 33.1.1.98 with many different Windows 10 64-bit devices with different cameras with Logitech Capture 1.10.110 or Logitech Capture 2.00.226 for Windows 10 (64-bit) installed.
Same problem in all cases with AIR 32-bit usage.
Other connected cameras doesn't matter. If Logitech Capture installed then AIR application crashes with any attempt to access to `Camera` APIs.
It works fine with AIR 64-bit.
It works fine with AIR 32-bit and 64-bit if no Logitech Capture software installed.

Related issues (not the same):
https://github.com/Gamua/Adobe-Runtime-Support/issues/53
https://github.com/Gamua/Adobe-Runtime-Support/issues/208

## Steps to Reproduce
1) Install Logitech Capture from official website https://www.logitech.com/en-us/product/capture to any Windows 10 device.
2) Launch code below on this device with bundled AIR 64-bit application. It just trace connected cameras names.

Application example with sources attached.
[camera_32bit_crash.zip](https://github.com/Gamua/Adobe-Runtime-Support/files/4596041/camera_32bit_crash.zip)

```
trace(Camera.names);
```

**Actual Result:**
Application crash.
```
Faulting application name: camera_32bit_crash.exe, version: 0.0.0.0, time stamp: 0x5e8d987b
Faulting module name: Adobe AIR.dll, version: 33.1.1.98, time stamp: 0x5e8da14b
Exception code: 0xc0000005
Fault offset: 0x0038f827
Faulting process id: 0x28c8
Faulting application start time: 0x01d624b9b2e042eb
Faulting application path: C:\camera_32bit_crash_32.app\camera_32bit_crash.exe
Faulting module path: C:\camera_32bit_crash_32.app\Adobe AIR\Versions\1.0\Adobe AIR.dll
Report Id: a8bcd9ce-e187-4278-add6-1f8e09db452a
Faulting package full name: 
Faulting package-relative application ID: 
```

**Expected Result:**
Application trace connected cameras names.
For example (as with 64-bit AIR usage) you will see in traces:
`PC Camera,Logi Capture`

## Known Workarounds
none
*Use 64-bit AIR version or deinstall Logitech Capture but it is not an option for our users and because of another AIR issue with 64-bit versions and Logitech cameras: https://github.com/Gamua/Adobe-Runtime-Support/issues/53

**Comment by ajwfrost** [comment]
FYI we have a fix for this one, will be included in our next release. Weird that it doesn't crash on the 32-bit version really! - it was some missing error handling that resulted in the crash.
thanks

**Comment by itlancer** [comment]
@ajwfrost, thank you for status. Does https://github.com/Gamua/Adobe-Runtime-Support/issues/53 will be fixed too?

**Comment by ajwfrost** [comment]
I suspect so, I've not got a Logitech camera here to find out and the crash log there wasn't from one of our builds so I can't tell for sure...
cheers

**Comment by itlancer** [comment]
It fixed with latest AIR 33.1.1.190. Thank you!
But related issue still not fixed: https://github.com/Gamua/Adobe-Runtime-Support/issues/53
